src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const currentPrice = PRICE_PER_DAY * days;
+  const SHORT_DISCOUNT = 20;
+  const LONG_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
```

Comment:
1) Move CONSTANTS outside the function.
CONSTANT - This is a variable that is created only 1 time and never changes.
In ur case, it is created every time the function is called
2) I think basePrice better describes this variable

```suggestion
  const basePrice = PRICE_PER_DAY * days;
```